types/parsimmon/parsimmon-tests.ts

```diff
@@ -284,6 +286,9 @@ strPar = P.seqMap(
 
 strArrPar = P.sepBy(P.string('foo'), P.string('bar'));
 strArrPar = P.sepBy1(P.string('foo'), P.string('bar'));
```

Comment:
Can we remove this line if we already have `str1ArrPar`?

Reply:
This line is helpful to ensure that the typing change to `P.sebPy1` is backwards-compatible, by ensuring that `Parser<[string, ...string[]]>` is still recognized as a valid subtype of `Parser<string[]>` by tsc. It should probably stay.

Reply:
Then everything looks good to me. :)